sourced/ml/algorithms/swivel.py

```diff
@@ -390,6 +390,10 @@ def main(_):
     tf.logging.set_verbosity(tf.logging.INFO)
     start_time = time.time()
 
+    omited = {"handler", "command"}
+    log("Swivel parameters:\n\t" + "\n\t".join(
+        "    {:20} {}".format(key, value)
+        for key, value in FLAGS.__dict__.items() if key not in omited))
```

Comment:
```python
 "\n".join("\t{:20} {}".format(key, value) for key, value in sorted(FLAGS.__dict__.items()) if key not in omited)))
```

BUT I saw that the number of epochs is printed as `float` is it a feature or a bug?

Reply:
feature, because:
```
n_total_steps = int(FLAGS.num_epochs * model.n_rows * model.n_cols) / (
            FLAGS.submatrix_rows * FLAGS.submatrix_cols)
```